ledger-core/testutils/predicate/predicate.go

```diff
@@ -9,25 +9,27 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/testutils/debuglogger"
 )
 
-func Never() Func {
+type Predicate = func(debuglogger.UpdateEvent) bool
```

Comment:
Do NOT name type same as package (except for very rare exceptions)

Reply:
done

Reply:
done